### ISSUE TITLE
Methods in a class are not evaluated correctly

### ISSUE THREAD

**Issue by AmeyWadodkar** [tntim96/JSCover#275]
This is my class
`class Person {
    constructor(name){
        this.name = name;
    };
    getname(){
        document.getElementById("name").innerHTML = this.name;
    };
}

var person = new Person("John Doe");
person.getname();`

and this is the instrumented file:
`_$jscoverage['/person.js'].lineData[1]++;
class Person {
  constructor(name) {
    _$jscoverage['/person.js'].functionData[0]++;
    _$jscoverage['/person.js'].lineData[3]++;
    this.name = name;
  }
  _$jscoverage['/person.js'].lineData[4]++;
  getname() {
    _$jscoverage['/person.js'].functionData[1]++;
    _$jscoverage['/person.js'].lineData[7]++;
    document.getElementById('name').innerHTML = this.name;
  }
  _$jscoverage['/person.js'].lineData[8]++;
}
_$jscoverage['/person.js'].lineData[11]++;
var person = new Person('John Doe');
_$jscoverage['/person.js'].lineData[12]++;
person.getname();
`
for :- _$jscoverage['/person.js'].lineData[4]++; eval() function gives an error SyntaxError: Unexpected token [

**Comment by tntim96** [comment]
Try removing the semi-colons after the constructor and method declarations:
```
class Person {
  constructor(name){
    this.name = name;
  }
  getname(){
    document.getElementById("name").innerHTML = this.name;
  }
}

var person = new Person("John Doe");
person.getname();
```

**Comment by tntim96** [comment]
Although this isn't a true bug, I've fixed this issue and uploaded a new 2.0.8-SNAPSHOT.
